honeybee_schema/energy/schedule.py

```diff
@@ -179,6 +179,17 @@ def check_end_date(cls, v):
         return cls.check_date(v)
 
 
+class ScheduleRule(ScheduleRuleAbridged):
+    """Schedule rule including a ScheduleDay and when it should be applied.."""
```

Comment:
We don't need this object since the `ScheduleRuleset` schema still uses `ScheduleRuleAbridged`. See here:
https://github.com/ladybug-tools/honeybee-energy/blob/master/honeybee_energy/schedule/ruleset.py#L582